src/styles/typography.scss

```
@@ -39,7 +39,7 @@ $body-font-sizes: (
 
 @mixin body-typography($body-name, $body-font-size) {
   .body#{$body-name} {
-    line-height: 1.3;
+    line-height: $body-font-size * 1.6;
```

Just using 1.6 in the line-height would be the same. Line heights expressed purely as [numbers](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#%3Cnumber%3E) are calculated by multiplying with the font-size. 
```suggestion
    line-height: 1.6;
```

---

Yup but if we use `rem` will be always the same value and that doesn't work for big font sizes.